chef/cookbooks/hyperv/recipes/windows_features.rb

```diff
@@ -8,27 +8,31 @@
     node.set[:windows_features_installed] = true
```

Comment:
it's just set to boolean only for hyperv as we are not doing anything with features in that case.